lib/CustomView/Config.php

```diff
@@ -52,14 +62,35 @@ private static function getRepository()
                 return null;
             };
 
+            $storageDirectory = null;
```

Comment:
Since this code block is used several times, we should think about making it generic available as a trait or so? 🤔